Workbooks/WorkbooksMetadata.json

```diff
@@ -797,5 +797,20 @@
     "templateRelativePath": "SysmonThreatHunting.json",
     "subtitle": "",
     "provider": "Azure Sentinel community"
+  },
+  {
+  "workbookKey": "CyberArkWorkbook",
+  "logoFileName": "CyberArk_Logo.svg",
+  "description": "The CyberArk Syslog connector allows you to easily connect all your CyberArk security solution logs with your Azure Sentinel, to view dashboards, create custom alerts, and improve investigation. Integration between CyberArk and Azure Sentinel makes use of the CEF Data Connector to properly parse and display CyberArk Syslog messages.",
+  "dataTypesDependencies": [ "CommonSecurityLog" ],
+  "dataConnectorsDependencies": [ "CyberArk" ],
+  "previewImagesFileNames": [ "CyberArk_White1.png", "CyberArk_Black1.png" ],
```

Comment:
Please check in the two preview files in the images folder per the workbook guidance https://aka.ms/sentinelgithubworkbooks